main/libpaper/template.py

```diff
@@ -1,32 +1,20 @@
 pkgname = "libpaper"
-pkgver = "1.1.29"
+pkgver = "2.2.5"
 pkgrel = 0
 build_style = "gnu_configure"
 make_cmd = "gmake"
 hostmakedepends = ["pkgconf", "gmake", "automake", "libtool", "gettext"]
 pkgdesc = "Library for handling paper characteristics"
 maintainer = "q66 <q66@chimera-linux.org>"
-license = "GPL-2.0-or-later"
-url = "http://packages.debian.org/unstable/source/libpaper"
-source = f"$(DEBIAN_SITE)/main/libp/{pkgname}/{pkgname}_{pkgver}.tar.gz"
-sha256 = "26330e21e9a3124658d515fd850b0cde546ff42d89b2596a5264c5f1677f0547"
+license = "MIT AND GPL-2.0-only AND GPL-3.0-or-later AND custom:none"
+url = "https://github.com/rrthomas/libpaper"
+source = f"{url}/releases/download/v{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "7be50974ce0df0c74e7587f10b04272cd53fd675cb6a1273ae1cc5c9cc9cab09"
 
 
 def post_install(self):
+    self.install_license("COPYING-MIT")
     self.install_dir("etc/libpaper.d", empty=True)
-    # systemwide default papersize
-    with (self.destdir / "etc/papersize").open("w") as ps:
-        ps.write("# Write the paper size here, see papersize(5)")
```

Comment:
this comment in `/etc/papersize` causes both `paperconf`and `paper` to bail out:
```
paper: unknown paper ‘# Write the paper size here, see papersize(5)’
```

Reply:
i'm not sure if i should remove old `/etc/papersize` files somehow as well to avoid this